README.MD

```
@@ -171,16 +183,42 @@ Below are instructions for installing the `olm` library for Linux (Debian and Ub
+   export PATH="/opt/homebrew/opt/grep/libexec/gnubin:$PATH"
+   ```
 
 ## Build
 
 Use the provided build script to build the application. This will create a binary in the `./build` directory.
-Minimum required go version is 1.24
+Minimum required go version is written in [go.mod](go.mod)
+
+We recommend not using global go installation but rather installing [gvm](https://github.com/moovweb/gvm) for easier go version management. After installing go, run the following command
 
+#### Linux
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)